lib/logdna.rb

```diff
@@ -119,15 +109,11 @@ def datetime_format(*_arg)
     end
 
     def close
-      if !@client.nil?
-        @client.exitout
-      end
+      @client&.exitout
```

Comment:
What does this operator mean here?

Reply:
`Rubocop` converted 3 lines into 1 line by using `&` operator

Reply:
ok I checked it is a safe navigation operator . I think this one is ok